internal/configs/testdata/error-files/module-sensitive.tf

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
do these error-files just get autmagically pulled in and checked that they error?